pkgs/by-name/mc/mcpat/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mcpat";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "HewlettPackard";
+    repo = "mcpat";
+    tag = "v${version}";
+    hash = "sha256-sr7H2vBOTyI59d3itVNqRVy1fR/83ZrTGl5s4I+g0Tw=";
+  };
+
+  postPatch = ''
+    substituteInPlace mcpat.mk \
+      --replace "CXX = g++ -m32" "CXX = g++" \
+      --replace "CC  = gcc -m32" "CC  = gcc"
+  ''
+  + lib.optionalString (!stdenv.hostPlatform.isx86) ''
+    substituteInPlace mcpat.mk \
+      --replace "-msse2 -mfpmath=sse" ""
+  '';
```

could you please use replace-fail instead

```suggestion
  postPatch = ''
    substituteInPlace mcpat.mk \
      --replace-fail "CXX = g++ -m32" "CXX = g++" \
      --replace-fail "CC  = gcc -m32" "CC  = gcc"
  ''
  + lib.optionalString (!stdenv.hostPlatform.isx86) ''
    substituteInPlace mcpat.mk \
      --replace-fail "-msse2 -mfpmath=sse" ""
  '';
```